spec/spec_helper.rb

```diff
@@ -1,7 +1,11 @@
+# frozen_string_literal: true
+
 require 'rspec'
+require 'pry'
```

Comment:
I'd remove this? People can add it back in if they need to pry debug on their own?